test/e2e/logs_test.go

```diff
@@ -535,6 +535,35 @@ var _ = Describe("Podman logs", func() {
 		Expect(logs).To(Not(Exit(0)))
 	})
 
+	It("podman logs with non ASCII log tag fails without env", func() {
+		oldLang := os.GetEnv("LANG")
+		os.UnsetEnv("LANG")
+		defer os.SetEnv("LANG", oldLang)
+
+		oldLC := os.GetEnv("LC_ALL")
+		os.UnsetEnv("LC_ALL")
+		defer os.SetEnv("LC_ALL", oldLC)
+
+		ctrName := "logsctr"
+		logc := podmanTest.Podman([]string{"run", "--name", ctrName, "-d", "--log-opt", "tag=\"äöüß\"", ALPINE, "echo", "podman"})
+		logc.WaitWithDefaultTimeout()
+		Expect(logc).To(Not(Exit(0)))
+	})
+
+	It("podman logs with non ASCII log tag succeeds", func() {
+		oldLang := os.GetEnv("LANG")
+		os.SetEnv("LANG", "en_US.UTF-8")
+		defer os.SetEnv("LANG", oldLang)
+
+		oldLC := os.GetEnv("LC_ALL")
+		os.SetEnv("LC_ALL", "")
+		defer os.SetEnv("LC_ALL", oldLC)
+
+		ctrName := "logsctr"
```

Comment:
pointless variable, could you actually just remove `"--name", ctrName, ` from lines 563 and 548, since container name is irrelevant?

And, you have to resubmit anyway because of `import os`.

Reply:
done!